modules/upgrade.nix

```diff
@@ -11,5 +11,11 @@
       createDevShm = ''
         mkdir -p ${config.build.installationDir}/dev/shm
       '';
+      createProcStat = ''
+        mkdir -p "${config.build.installationDir}/proc"
+        if [ ! -e "${config.build.installationDir}/proc/stat-fake" ]; then
+          echo "btime 0" > "${config.build.installationDir}/proc/stat-fake"
```

Comment:
I'm wary of managing content that way, working around Nix. What if we'll want to later extend it with more fields for some other software?

What about taking a different approach: make it a config file in `/etc` by adding one more module to `modules/environment`? That way there will be no need to create `/proc` and no upgrade concerns, plus configurability and overrideability through Nix. And in login we can check for "unreadable real file *and* present fake file => bind-mount it as /proc/stat". Does that sound reasonable?

One quirk I foresee though is that this `/etc/proc-stat-fake` will be a symlink that doesn't make sense outside of proot, so getting to the file we want to bind-mount will require prefixing, something like `"$installationDir/$(readlink $installationDir/etc/proc-stat-fake)"`.